pyomo/util/calc_var_value.py

```diff
@@ -190,7 +195,13 @@ def calculate_variable_from_constraint(variable, constraint,
                 "expression.\n\tPlease provide a different initial guess "
                 "or enable the linesearch if you have not.")
             raise
-        fpk = value(expr_deriv)
+
+        if diff_mode in {differentiate.Modes.sympy, differentiate.Modes.reverse_symbolic}:
```

Comment:
Note that no test covers the case that `reverse_symbolic` is specified....

Reply:
Done!